src/js/components/DataFilters/DataFilters.js

```diff
@@ -49,7 +49,7 @@ export const DataFilters = ({ drop, children, heading, layer, ...rest }) => {
   );
 
   const clearControl = badge && (
-    <Box flex={false}>
+    <Box flex={false} margin={{ start: 'small' }}>
```

Comment:
Not sure where to comment on it, but the gap between the layer footer buttons is too large. In HPE patterns, we use gap="small"

Reply:
FYI, the default `gap` in Footer is "medium". Explicitly made it "small" for DataForm.